packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx

```diff
@@ -22,7 +22,7 @@ import { expect } from "@storybook/jest";
 import { axeRun } from "../../../axe-util";
 import type { PlaywrightProps } from "../storybookTypes";
 
-const storybookMeta: Meta<CheckboxProps> = {
+const storybookMeta: Meta<typeof Checkbox> = {
```

Comment:
I thought using `CheckboxProps` was better for these. Any reason we're using `typeof Checkbox` now?

I know Storybook v7 changed some TypeScript pieces, so I'm thinking you can elaborate.

Reply:
I forgot to roll it back, jus t was playing around the code a little bit

Reply:
Updated